infra/containers.yaml

```diff
@@ -147,6 +147,8 @@ Resources:
               Value: !FindInMap [AdminGroup, !Ref Environment, group]
             - Name: BASE_URL
               Value: !Sub "https://${DomainName}"
+            - Name: MEASUREMENT_ID
+              Value: !Sub "https://${MeasurementId}"
```

Comment:
You need to add `MeasurementId` to the parameters here at the top of the file.  Follow the example of `DomainName` maybe?

Reply:
You'll also need to add it to the main yaml file (along with the default, there), and pass it in - like `DomainName`.

Reply:
@ndmckinley thanks.  Which is the main yaml file?